lib/types/lvm_vg.nix

```diff
@@ -33,6 +38,16 @@
             default = [ ];
             description = "Extra arguments";
           };
+          pool = lib.mkOption {
```

Comment:
isn't this the same as config.name? btw I refactored this mode a little bit in https://github.com/nix-community/disko/pull/284

Reply:
i don't think so. this is used for layered LVs. here I'm using it for a thin lv, on a thinpool (thinmeta+thindata LV). it could also be used for vdo, for example.

if you look at how it's used in the `lvcreate` invocation, this is passed to `--thinpool=`.